### PR TITLE
SISRP-52605 - Entry Into Alum/ Former Portal

### PR DESCRIPTION
**Pull request by nkutub**
https://jira-secure.berkeley.edu/browse/SISRP-52605

* sign out of CLC 
* fix nav showing after navigation

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -29,40 +30,44 @@
...
+  <noscript>
+    <meta http-equiv="Refresh" content="0; URL=/enable-javascript.html"></noscript>
 
   <base href="/">
 
-  <link href="https://fonts.googleapis.com/css?family=Lato:300,300i,400&amp;subset=latin-ext"
-        rel="stylesheet"
-        data-ng-if="isBcourses">
+  <link href="https://fonts.googleapis.com/css?family=Lato:300,300i,400&amp;subset=latin-ext" rel="stylesheet"
+    data-ng-if="isBcourses">
```
Comment (redconfetti) [inline src/index.html:39]: @nkutub - Maybe we should get rid of this line given that we don't support bCourses / Canvas?

File: src/index.html [hunk 2]
```diff
@@ -71,9 +76,12 @@
...
+
+  <div class="footer-sticky-container" data-ng-if="(controllerName === 'SplashController')"
+    data-ng-include="'footer.html'"></div>
   <div data-ng-include="'acting_as.html'"></div>
   <div data-ng-include="'advisor_acting_as.html'"></div>
   <div data-ng-include="'delegate_acting_as.html'"></div>
 </body>
-</html>
+
+</html>
```
Comment (redconfetti) [inline src/index.html:87]: @nkutub  - Can we add a newline at the end of the file. I know it's not a functional issue, but it's some sort of UNIX POSIX requirement for text files.

Comment (nkutub) [reply src/index.html]: I did add it not showing in gitHub

File: src/index.html (hunk 1 above)
Comment (nkutub) [reply src/index.html]: removed

**Pull request merged**